### ISSUE TITLE
False positive with `Style/FormatStringToken` in `routes.rb`

### ISSUE THREAD

**Issue by anthony-robin** [rubocop-hq/rubocop#7452]
I'm using some variable as `template` style in my route redirection and Rubocop
flag an offense on it:

```ruby
# config/routes.rb

get '/catalogue/:book_id/book', to: redirect('book/%{book_id}', status: 301)
# => Prefer annotated tokens (like `%<foo>s`) over template tokens (like `%{foo}`)
```

<img width="952" alt="Capture d’écran 2019-10-22 à 11 48 31" src="https://user-images.githubusercontent.com/5428510/67275252-7b3def00-f4c2-11e9-9af0-df4ef07f4fb3.png">

There should not be offense flagged because using the `annotated` syntax, it throws an error:

```
bad URI(is not URI?): book/%<book_id>s
```

<img width="487" alt="Capture d’écran 2019-10-22 à 11 53 59" src="https://user-images.githubusercontent.com/5428510/67275333-a9233380-f4c2-11e9-989f-dd7e42c60023.png">

Should the `redirect` method be excluded from the checked methods or the cop disabled for `routes.rb` ?

## RuboCop version

```
$ [bundle exec] rubocop -V
0.75.1 (using Parser 2.6.5.0, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by coneybeare** [comment]
I agree this is a false positive.
